pkg/util/k8sutil/backup.go

```diff
@@ -45,6 +45,8 @@ const (
 	awsSecretVolName          = "secret-aws"
 	awsConfigVolName          = "config-aws"
 	fromDirMountDir           = "/mnt/backup/from"
+
+	backupDir = "/var/etcd-operator-backup/v1"
```

Comment:
no util please. util should contain utility functions. not shared types or constants.